.github/pre-commit-config.yaml

```diff
@@ -9,19 +14,7 @@ repos:
         pass_filenames: false
       - id: prettier
         name: Prettier
-        entry: pnpm
-        args:
-          [
-            "--dir",
-            ".github",
-            "exec",
-            "prettier",
-            "--ignore-path",
-            "../.prettierignore",
-            "--check",
-            "--ignore-unknown",
-            "../",
-          ]
+        entry: bash -c 'cd .github && git -C .. ls-files "*.md" "*.json" "*.yaml" "*.yml" "*.css" "*.js" "*.ts" "*.tsx" | sed "s|^|../|" | xargs pnpm exec prettier --ignore-path ../.prettierignore --check --ignore-unknown --no-error-on-unmatched-pattern'
```

Comment:
Why are we `cd`ing here only to then give `git` the parent directory as the context?

Reply:
prettier is only installed in the subdirectory as an pnpm package.... I'll deal with this in a follow-up much more cleanly!